src/lib/Repository/UiTranslationService.php

```diff
@@ -33,11 +33,8 @@ public function __construct(LanguageService $languageService)
      */
     public function loadTranslations(VersionInfo $versionInfo)
     {
-        $translations = [];
-        foreach ($versionInfo->languageCodes as $languageCode) {
-            $translations[] = $this->languageService->loadLanguage($languageCode);
-        }
-
-        return $translations;
+        return array_map(function ($languageCode) {
```

Comment:
I personally find this syntax to be easier to read:
```
return array_map(
  function ($languageCode) {
    return $this->languageService->loadLanguage($languageCode);
  },
  $versionInfo->languageCodes
```

Not a request for change though.